lib/PuppeteerSharp/ResponseData.cs

```diff
@@ -39,5 +39,10 @@ public string Body
         /// </summary>
         /// <value>Status Code.</value>
         public HttpStatusCode? Status { get; set; }
+        /// <summary>
+        /// Multiple Response headers with same Header name. Header values will be converted to a string.
+        /// </summary>
+        /// <value>Headers.</value>
+        public List<KeyValuePair<string, object>> MultipleHeaders { get; set; }
```

Comment:
Using [NameValueCollection](https://docs.microsoft.com/en-us/dotnet/api/system.collections.specialized.namevaluecollection?view=netstandard-2.0) would likely be nicer here, it supports multiple values which can be obtained via `GetValues`. It's part of `.Net Standard 2.0`.

Reply:
Using https://docs.microsoft.com/en-us/dotnet/api/system.net.http.headers.httpheaders?view=net-5.0 would be even better if it's possible.